corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/fake_formplayer.js

```diff
@@ -1,185 +1,167 @@
-/**
- * Generates mock formplayer responses. Use as a fake for queryFormplayer.
- *
- * This does not support many of the options that can be passed to the real queryFormplayer.
- * It primarily fakes the selections logic.
- *
- * This contains one app, which has the following structure:
- *      m0: a menu that does not use cases
- *          m0-f0
- *      m1: a menu that uses cases
- *          m1-f0: a form that updates a case
- *  No menus use display-only forms.
- */
```

Comment:
Thinking we probably want to keep this in here.

Reply:
Indeed, thank you. Fixed in 03ea0b0b1ee, and 760fb698fe3f078b12c3a4bd92bedd0db3f23407 updates the script so this doesn't happen again.